Add missing URI separator in file URIs

henrikt-ma opened the pull request:
Fixes #3610

beutlich commented:
I rebased on master and resolved the merge conflicts. Additionally, I also added the third slash to the other occurrences.

henrikt-ma commented:
> I rebased on master and resolved the merge conflicts.

Oh, sorry, my bad.  I based the work on the stone age _master_ in my fork instead of the one in the central repo.

> Additionally, I also added the third slash to the other occurrences.

Excellent!
